<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTableWidgetTp</class>
 <widget class="QWidget" name="QTableWidgetTp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_Icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>541</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>590</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Line">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>541</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_iconTpye">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addIcon">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>addIcon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> IconType</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Height</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_width">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>70</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_height">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>70</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_nodeS">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_nodeE">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> End</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addLine">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>addLine</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
